modules/ccs-aws-iam.adoc

```diff
@@ -107,7 +107,11 @@ The `osdManagedAdmin` user is created immediately after taking control of the cu
 
 ** AmazonEC2ReadOnlyAccess
 ** CustomerAdministratorAccess
-
++
+[NOTE]
+====
+The `network-mgmt` role only applies to non-Customer Cloud Subscription (CCS) clusters.
+====
```

Comment:
The writing looks good to me, but I don't think this needs to be in a [NOTE]. What do you think about adding it to line 106 before the sentence "The following policies are attached to the role:"? The way it's formatted now, it looks like the note only applies to the `CustomerAdministratorAccess` role.

Reply:
Thanks so much! I made the changes. Please let me know if there is anything else you find.